### ISSUE TITLE
Calendar button enhancements

### ISSUE THREAD

**Issue by nstringham** [kendoclub-at-umich/website#202]
Replace the built-in full calendar buttons with pico buttons. This will require quite a bit of CSS to get looking right but it should be look better than what we currently have and it should be better for accessibility/SEO because we can correctly use `<a>` and `<button>`.
